src/backend/aspen/app/views/phylo_trees.py

```diff
@@ -64,30 +70,33 @@ def phylo_trees():
                 phylo_run_alias.group_id.in_(cansee_owner_group_ids),
             )
         )
-        .filter(phylo_run_alias.workflow_status == WorkflowStatusType.COMPLETED)
     )
 
     # filter for only information we need in sample table view
     results: MutableSequence[Mapping[str, Any]] = list()
     for phylo_run in phylo_runs:
-        phylo_tree: PhyloTree
+        phylo_tree: Optional[PhyloTree] = None
+        result: Mapping[str, Any] = {}
         for output in phylo_run.outputs:
             if isinstance(output, PhyloTree):
                 phylo_tree = output
-                break
-        else:
-            raise RecoverableError(
-                f"phylo run (workflow id={phylo_run.workflow_id}) does not have a"
-                " phylo tree output."
-            )
-        results.append(
-            {
-                "phylo_tree_id": phylo_tree.entity_id,
-                "name": humanize_tree_name(phylo_tree.s3_key),
-                "pathogen_genome_count": 0,  # Leaving this field in place temporarily for reverse-compatibility
-                "completed_date": format_datetime(phylo_run.end_datetime),
+                result = result | {
+                    "phylo_tree_id": phylo_tree.entity_id,
+                    "name": phylo_tree.name or humanize_tree_name(phylo_tree.s3_key),
+                }
+        if not phylo_tree:
+            result = result | {
+                "phylo_tree_id": None,
+                "name": phylo_run.name or generate_tree_name_from_template(phylo_run),
             }
-        )
+        result = result | {
```

Comment:
Can we make this the default result instead of {} on line 79 instead of merging it in here?